osdk-core/modules/src/filemgr/impl/dji_file_mgr_impl.cpp

```diff
@@ -593,10 +593,10 @@ FilePackage FileMgrImpl::parseFileList(std::list<DataPointer> fullDataList) {
   /*! Parse file list total info */
   auto listdata = (dji_file_list_download_resp *)((uint8_t *)dataFront.data + sizeof(dji_general_transfer_msg_ack) - 1);
   DSTATUS("###data->amount = %d, data->len = %d", listdata->amount, listdata->len);
-  if (listdata->len > fullDataList.size() * maxDataFrameLen) {
-    DERROR("File list total data len error !");
-    return pack;
-  }
+  // if (listdata->len > fullDataList.size() * maxDataFrameLen) {
+  //   DERROR("File list total data len error !");
+  //   return pack;
+  // }
```

Comment:
Is there any possible issues that disabling this check can cause? 

I'm guessing part of the issue is with how maxDataFrameLen is defined as 1024, not allowing for large files like videos to be listed in this manner without disabling the check?

Reply:
Actually this is suggested fix solution from DJI.  Also I just feel the len check here does not make sense.

Reply:
I agree that the length-check that was initially present within the Onboard-SDK didn't make sense, since camera-files can be of type=video (variable length) or a picture(typically the same size for the same sensor). It seems like that part of the OSDK was developed with image-capture in mind.